src/quo/components/utilities/token/loader.cljs

```diff
@@ -6,9 +6,10 @@
 
 (defn- get-token-image*
   [token]
-  (let [token-symbol (cond-> token
-                       (keyword? token) name
-                       :always          (comp string/lower-case str))]
-    (get tokens token-symbol)))
+  (when token
+    (let [token-symbol (cond-> token
+                         (keyword? token) name
+                         :always          string/lower-case)]
+      (get tokens token-symbol))))
```

Comment:
Not related to this PR, recently @J-Son89 and I noticed token images were broken so we are fixing this. Probably @J-Son89 's PR will be merged first, so this change is not important